scripts/doctor.py

```diff
@@ -48,10 +48,10 @@ def main():
       gcloud=check_gcloud,
       gcloud_login=(check_gcloud_login, ['gcloud']),
       gcloud_project=(check_gcloud_default_project, ['gcloud_login']),
+      gsutil=check_gsutil,
```

Comment:
when I said after, I meant to include gcloud_loging as a dependency. The order here doesn't matter. This should be written as:

```python
gsutil=(check_gsutil, ['gcloud_login'])
```

Reply:
https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/pull/375